code/modules/power/tesla/energy_ball.dm

```diff
@@ -81,72 +83,78 @@
 			var/list/temp_shock = list()
 			tesla_zap(ball, range, TESLA_MINI_POWER/7*range, shocked_targets = temp_shock)
 			shocked_things += temp_shock
-	else
-		energy = 0 // ensure we dont have miniballs of miniballs //But it'll be cool broooooooooooooooo
 
-/obj/singularity/energy_ball/examine(mob/user)
+/obj/energy_ball/examine(mob/user)
 	. = ..()
 	if(orbiting_balls.len)
 		. += "There are [orbiting_balls.len] mini-balls orbiting it."
 
-
-/obj/singularity/energy_ball/proc/move_the_basket_ball(move_amount)
+/obj/energy_ball/proc/move(move_amount)
 	var/list/dirs = GLOB.alldirs.Copy()
-	for(var/I in 1 to 30)
+	for (var/_ in 1 to 30)
```

Comment:
this is somehow even less helpful of a var name

Reply:
its completely unused, so it doesnt get a name

Reply:
I is semi standard, and I think it's fine enough here ~~says the man who wrote this shitcode fuck I need to touch this up~~